cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,92 @@
 /// <reference types='cypress' />
+const { generateUser } = require('../support/generateUser');
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
+  const { firstName, lastName, email, age, salary, department } =
+  generateUser();
 
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should have pagination', () => {
+    cy.get('.-pagination').should('exist');
+    cy.contains('.-next > .-btn', 'Next').should('exist');
+    cy.contains('.-previous > .-btn', 'Previous').should('exist');
+  });
+
+  it('should have rows count selection', () => {
+    cy.get('.-center').should('exist');
+    cy.get('select').select('5 rows');
+    cy.get('select').should('contain', '5 rows');
+  });
+
+  it('should add a new employee', () => {
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(firstName);
+    cy.get('#lastName').type(lastName);
+    cy.get('#userEmail').type(email);
+    cy.get('#age').type(age);
+    cy.get('#salary').type(salary);
+    cy.get('#department').type(department);
+    cy.get('#submit').click();
+    cy.get('.rt-td').should('contain', firstName);
+    cy.get('.rt-td').should('contain', lastName);
+    cy.get('.rt-td').should('contain', email);
+  });
+
+  it('should delete employee', () => {
+    cy.get('#delete-record-1').click();
+    cy.get('.rt-td').should('not.contain', email);
```

Comment:
The same about all columns...

Reply:
It's better to check that this user no longer exists at all, not just email

Reply:
done